ddtrace/internal/ci_visibility/writer.py

```diff
@@ -105,7 +105,6 @@ def __init__(
         dogstatsd=None,  # type: Optional[DogStatsd]
         sync_mode=False,  # type: bool
         report_metrics=False,  # type: bool
-        api_version=None,  # type: Optional[str]
```

Comment:
This argument is never used. I can revert this but just wanted to highlight this quirk.